planning/behavior_path_planner/autoware_behavior_path_goal_planner_module/src/goal_planner_module.cpp

```diff
@@ -670,6 +659,7 @@ void GoalPlannerModule::updateData()
     goal_candidates_ = generateGoalCandidates();
   }
 
+  occupancy_grid_map_->setMap(*(planner_data_->occupancy_grid));
```

Comment:
memo for me.
it is ok not to lock becauses this is used in only main thread.

Reply:
currently it is used in isStuck(), but will remove that

Reply:
by doing
- remove occupancy_grid from isStuck()
- do goal_searcher_->update() in updateData() and passed to lane_parking/freespace_parking thread.

we do not need to share occupnacy grid map.